packages/components/tooltip/src/tooltip.visualroute.jsx

```diff
@@ -45,6 +46,31 @@ export const component = () => {
           </Tooltip>
         </ContainerWithPadding>
       </Spec>
+      <Spec
+        label="CollapsiblePanel as a parent"
+        listPropsOfNestedChild={true}
+      >
+        <CollapsiblePanel
```

Comment:
The `CollapsablePanel` component has css property `white-space: nowrap` so in this test we can make sure that it doesn't override the tooltip's `white-space: break-spaces`. 

<img width="330" alt="Screenshot 2023-09-25 at 14 48 03" src="https://github.com/commercetools/ui-kit/assets/52276952/c0f4dcf1-3abe-482f-afe4-63f4fccbda2e">